### ISSUE TITLE
write, alert debug etc. not working 

### ISSUE THREAD

**Issue by mmhiguera** [googleapis/nodejs-logging#78]
Only method delete is working on the next envionment

#### Environment details

  - OS: Linux
  - Node.js version: v8.5.0
  - npm version: 5.3.0
  - @google-cloud/logging version: 1.1.4

However, same code runs perfectly on Windows environment ( node version 8.9.1 npm version 5.6.0 and loggin version 1.1.4 )

#### Steps to reproduce

  1. log.write (or debug, alert, etc)

**Comment by stephenplusplus** [comment]
I'm trying to reproduce, but cannot. Are you receiving an error message?

**Comment by mmhiguera** [comment]
That's the problem, I am not, it's just like the code does not execute but I don't receive neither message of operation successful nor error message from the request. It's unusual because in fact I can use delete method and it works (only this one it works)

**Comment by stephenplusplus** [comment]
Thank you. Are you running the sample from the command line?

**Comment by mmhiguera** [comment]
Yes, I am running "node quickstart.js" from samples directory

**Comment by mmhiguera** [comment]
I found the problem but it makes no sense and I don't know how to solve it. When I run the code on Windows the enviroment shows next vars:
{ IS_APP_ENGINE: false,
  IS_CLOUD_FUNCTION: false,
  **IS_COMPUTE_ENGINE: false**,
  IS_CONTAINER_ENGINE: false }

...but when I run the same code on Linux the result of environment is:
{ IS_APP_ENGINE: false,
  IS_CLOUD_FUNCTION: false,
  **IS_COMPUTE_ENGINE: true**,
  IS_CONTAINER_ENGINE: false }

I realize that if I change manually the content of environment the code runs perfectly and I get the same results that If I were running the code on Windows.

Any help would be appreciated.

Thanks in advance.

**Comment by stephenplusplus** [comment]
That is strange. `IS_COMPUTE_ENGINE` is determined from this function: https://github.com/stephenplusplus/google-auto-auth/blob/765c8a0e85184538bda25f33bb949aa842eeff23/index.js#L217

The check is:

```js
request('http://metadata.google.internal', (err, res) => {
  env.IS_COMPUTE_ENGINE = !err && res.headers['metadata-flavor'] === 'Google';
  callback(null, env.IS_COMPUTE_ENGINE);
});
```

So, it seems that somehow, the Linux environment you're in is able to respond to that request, and it has the appropriate headers. Is it possible an emulator is running, or any other tooling layer that might be different in the Linux stack?

**Comment by mmhiguera** [comment]
I have run the code from a server in Compute Engine and from Cloud Shell

**Comment by stephenplusplus** [comment]
Would you mind explaining more complete reproduction steps? I'm uncertain of the environments you're running in, and what you're experiencing.

**Comment by mmhiguera** [comment]
I am running code on Compute Engine Instance :
Node.js version: v8.5.0
npm version: 5.3.0
version gcloud logging: 1.1.4

1. Clone this repo https://github.com/googleapis/nodejs-logging
2. Install the client library with **npm install --save @google-cloud/logging**
3. Go to **samples** directory and run quickstart.js with command `node quicktstart.js`
4. Get no error neither any info but log entry does not save int stackdriver logging

That's all info I can give you besides the one you already have.

I hope this helps. 

Thanks

**Comment by stephenplusplus** [comment]
Thank you. And your discovery about `IS_COMPUTE_ENGINE`, when you said:

> When I run the code on Windows the enviroment shows next vars:
> { IS_APP_ENGINE: false,
> IS_CLOUD_FUNCTION: false,
> IS_COMPUTE_ENGINE: false,
> IS_CONTAINER_ENGINE: false }

Do you mean within a Windows Compute Engine instance, `IS_COMPUTE_ENGINE` returns false?

Overall, I'm confused about that discovery that you made, so anything to elaborate would be appreciated. Thanks!

**Comment by mmhiguera** [comment]
No, the Windows instance I use is not a Compute Engine instance, what I mean is that it seems there is a problem when it's got **compute_engine: true** because if I modify this param manually, the code runs perfectly. Hope I explain myself better :)

**Comment by Cshion** [comment]
I was running same issue, i think the problem is with 1.1.4, im using nodejs-logging-winston. When I install this package on a compute instance running ubuntu 16.04 it install by default 1.1.4, so I had to manually install first nodejs-logging 1.1.2 then it works.

**Comment by mwilliammyers** [comment]
I am having the same issue.

**Comment by mmhiguera** [comment]
Perfect !! It works with 1.1.2. 

Thanks for your help.

**Comment by stephenplusplus** [comment]
I believe the fix has just been released under 1.1.5. Please try it out and let me know how it goes!
